samples/nrf9160/modem_trace_flash/prj.conf

```diff
@@ -25,11 +29,12 @@ CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
 CONFIG_NRF_MODEM_LIB_TRACE=y
 CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_FLASH=y
 
-# Stream flash
-CONFIG_STREAM_FLASH=y
+# Flash config
+CONFIG_FCB=y
 CONFIG_FLASH=y
 CONFIG_FLASH_MAP=y
 
+CONFIG_NRF_MODEM_TRACE_FLASH_NOSPACE_SIGNAL=y
```

Comment:
does it make sense for this sample to choose this option, since it does not show how to handle that event?

Reply:
if we want to keep that here, we should show how to handle it and how to set up the callback (and when).

Reply:
I think it would be useful to show what should be done (log something in the callback) when `CONFIG_NRF_MODEM_TRACE_FLASH_NOSPACE_SIGNAL` is set because the other option (`CONFIG_NRF_MODEM_TRACE_FLASH_NOSPACE_ERASE_OLDEST`) doesn't require anything from the application.

Reply:
I'll change to `ERASE_OLDEST` for now. Might add it later.